src/headless/core.js

```diff
@@ -998,6 +998,9 @@ function initPersistentStorage () {
     } else if (_converse.api.settings.get("persistent_store") === 'IndexedDB') {
         config['description'] = 'indexedDB instance';
         config['driver'] = [Storage.localForage.INDEXEDDB];
+    } else if (_converse.api.settings.get("persistent_store") === 'in_memory') {
```

Comment:
I prefer `none` to `in_memory` for the config setting value.

It's more semantic because strictly speaking, there is no persistent storage in this case.

Reply:
Makes sense :)